src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -183,6 +182,10 @@ def run_sequential_stages(
                     "the stage name."
                 )
 
+            # skip stages which have already been applied
```

Comment:
This does not actually affect this example, but is a good change to make. This guards against applying the same stage multiple times.